microsoftgraph/graph-powershell-1.0/Microsoft.Graph/Add-MgEnvironment.md

```diff
@@ -0,0 +1,129 @@
+﻿---
+external help file: Microsoft.Graph.Authentication.dll-Help.xml
+Module Name: Microsoft.Graph.Authentication
+online version: https://learn.microsoft.com/en-us/powershell/module/microsoft.graph.authentication/add-mgenvironment
```

Comment:
Why do we have duplicate help files under `graph-powershell-1.0/Microsoft.Graph.Authentication` and `graph-powershell-1.0/Microsoft.Graph`?

Reply:
Thanks for noticing this. I have made the corrected the error